src/gpuhunt/_internal/utils.py

```diff
@@ -16,3 +16,10 @@ def parse_compute_capability(
         major, minor = value.split(".")
         return int(major), int(minor)
     return value
+
+
+def to_camel_case(snake_case: str) -> str:
+    words = snake_case.split("_")
+    words = list(filter(None, words))
+    words[1:] = [word[:1].upper() + word[1:] for word in words[1:]]
```

Comment:
Is it possible to use `str.title()` and `str.capitalize()` functions here?

Reply:
They would convert all letters but the first to lowercase. This is not important for our use case, but I thought keeping the existing case is more intuitive.

E.g. `spamHam_eggs` would become `spamhamEggs`, while with the current implementation it becomes `spamHamEggs`.

Reply:
Yes, thanks!